client/docs/websites.md

```diff
@@ -0,0 +1,34 @@
+# Monorepo: Websites
+
+This monorepo has special support for website packages, in the `web/` folder. These are quite a bit more complex than module packages. While technically there is no specific restriction on what can be used to build or develop a website package, it is recommended that you use [Snowpack](https://www.snowpack.dev/). This doc. will assume that you're using Snowpack and that you started with the `templates/website-template` configuration.
+
+The file structure will look like this:
+```
+website:
+- public/ (might have to be created)
+  - **/*
+- src/
+  - index.html
+  - **/*
+- package.json
+- snowpack.config.cjs
+- tsconfig.json
+```
+
+The `public` folder is where source files that do not need compiling or transformations by Snowpack are kept. The `src` folder is where the rest of the source files go, such as TypeScript, CSS, etc. The file structure of these two folders are compiled/built into a `dist/` folder by Snowpack.
+
+Other than the inheritance of development dependencies and tooling configurations, websites act like fairly normal NPM packages. If you need to add a dependency to one, you can either navigate to the website package and do a normal `npm i <dep>` command, or manually add it to the `package.json`. After you do that, you need to make sure to run the `npm run bootstrap` command at the root. This ensures that all the fragile symlinks get restored by Lerna.
+
+If you are wondering how you actually make a website using Snowpack, I would look into the Snowpack documentation. It's a fairly straightforward process.
```

Comment:
One thing I'm a little confused about with this is what "website" means here. This is for building the Wikijump frontend, why would there be websites within our website?

Reply:
`web/` holds any number of websites. Wikijump already has at least two - the `.org` blog, and the actual wiki. One of these website packages would be the frontend proper, maybe just named `frontend`.

Reply:
Oh I see, okay.